libkbfs/folder_branch_ops.go

```diff
@@ -1391,18 +1391,18 @@ func (fbo *folderBranchOps) SetInitialHeadFromServer(
 // object and sets the head to that.
 func (fbo *folderBranchOps) SetInitialHeadToNew(
 	ctx context.Context, id tlf.ID, handle *TlfHandle) (err error) {
-	fbo.log.CDebugf(ctx, "SetInitialHeadToNew")
-	defer func() {
-		fbo.deferLog.CDebugf(ctx, "Done: %v", err)
-	}()
-
 	rmd, err := makeInitialRootMetadata(
 		fbo.config.MetadataVersion(), id, handle)
 	if err != nil {
 		return err
 	}
 
 	return runUnlessCanceled(ctx, func() error {
+		fbo.log.CDebugf(ctx, "SetInitialHeadToNew")
```

Comment:
Moot, but I did it because otherwise the line number in the defered log was useless.